packages/gatsby-theme-store/src/sdk/pixel/events.ts

```diff
@@ -60,7 +59,7 @@ export type OrderPlacedData = Order
 export type OrderPlacedTrackedData = Order
 
 export interface ProductViewData {
-  product: ServerProductPageQueryQuery['product']
+  product: any
```

Comment:
This should be any? Where is the old type?

Reply:
The query doesn't exists anymore, so the type isn't being created. We can't replicate this type though, because this type structure (from query) can be changed inside the store.

I don't like this any but idk if we have a way to handle this haha.

Reply:
since we removed the query this type doesn't really exist anymore